charts/rhtap-infrastructure/values.yaml

```diff
@@ -165,3 +171,29 @@ infrastructure:
             pg_hba:
               - host all all 0.0.0.0/0 scram-sha-256
               - host all all ::1/128 scram-sha-256
+  #
+  # OpenShift Pipelines
+  #
+  openShiftPipelines:
+    # Toggles the settings related to OpenShift Pipelines components.
+    enabled: false
+    # The primary OpenShift Pipelines instance name.
+    name: tekton-pipeline
+    # The namespace where the OpenShift Pipelines components is installed.
+    namespace: openshift-pipelines
+    # Patches the global TektonConfig resource.
+    patchClusterTektonConfig:
+      # Resource name to patch.
+      resourceName: config
+      # Annotations to add to the resource.
+      annotations: {}
+      # Labels to add to the resource.
+      labels: {}
+    # Tekton Chains settings.
+    tektonChains:
```

Comment:
This section can go away, I don't see it's value.

Reply:
That section controls Tekton Chains related settings, and replaces hook scripts. In fact, is quite relevant.

Reply:
I don't think any of these values are controlled by the user.

Reply:
These attributes have defaults, they should be good enough to work with the current version of OpenShift Pipelines, and allow us to modify as we move forward. The entries added to the `values.yaml` are, in fact, variables that we use in multiple resources to avoid duplication, among other reasons to define them.

Additionally, by having these attributes we can leverage their relationship via `values.yaml.tpl`.

Reply:
For the record, I moved the defaults to the `values.yaml` on `rhtap-infrastructure` instead of setting them up on `values.yaml.tpl`.